src/store/Chat/index.js

```diff
@@ -218,6 +227,7 @@ function ChatReducer (state = initialState, action) {
       return state;
 
     case SET_ACTIVE_CHAT:
+      console.log('SET ACTIVE HAT ACTION IS', action)
```

Comment:
Also probably dont _need_ this console log either?

Reply:
yeah. sanity checks.